tests/src/Functional/DeveloperAppUITestTrait.php

```diff
@@ -133,7 +137,14 @@ protected function postCreateAppForm(array $data, ?UserInterface $account = NULL
       $account = $this->account;
     }
 
-    $this->drupalPostForm("/user/{$account->id()}/apps/create", $data, 'Add developer app');
+    $this->drupalPostForm(
```

Comment:
Please add a test that ensures the custom breadcrumb is correct on the Add app form, this test should also validate if the custom breadcrumb works properly for different users (breadcrumb cache issue).